doc/source/reference/random/new-or-different.rst

```diff
@@ -9,38 +9,40 @@ NumPy 1.17.0 introduced `Generator` as an improved replacement for
-                                        with any BitGenerator.
------------------- -------------------- -------------
-``random``         ``random_sample``,   Access the values in a BitGenerator,
-                   ``rand``             convert them to ``float64`` in the
-                                        interval ``[0.0.,`` `` 1.0)``.
-                                        In addition to the ``size`` kwarg, now
-                                        supports ``dtype='d'`` or ``dtype='f'``,
-                                        and an ``out`` kwarg to fill a user-
-                                        supplied array.
-
-                                        Many other distributions are also
-                                        supported.
------------------- -------------------- -------------
-``integers``       ``randint``,         Use the ``endpoint`` kwarg to adjust
-                   ``random_integers``  the inclusion or exclusion of the
-                                        ``high`` interval endpoint
-================== ==================== =============
+======================= ================== =============
+Feature                 Older Equivalent   Notes
+----------------------- ------------------ -------------
+`Generator`             `RandomState`      `Generator` requires a stream
+                                           source, called a `BitGenerator`
+                                           A number of these are provided.
+                                           `RandomState` uses the Mersenne
+                                           Twister `MT19937` by default,
+                                           but can also be instantiated
+                                           with any BitGenerator.
+----------------------- ------------------ -------------
+`~.Generator.random`    `random_sample`,   Access the values in a
+                        `rand`             BitGenerator, convert them to
+                                           ``float64`` in the interval
+                                           ``[0.0.,`` `` 1.0)``. In addition
+                                           to the ``size`` kwarg, now
+                                           supports ``dtype='d'`` or
+                                           ``dtype='f'``, and an ``out``
+                                           kwarg to fill a user-supplied
+                                           array.
+
+                                           Many other distributions are also
+                                           supported.
+----------------------- ------------------ -------------
+`~.Generator.integers`  `randint`,         Use the ``endpoint`` kwarg to
+                        `random_integers`  adjust the inclusion or exclusion
+                                           of the ``high`` interval endpoint.
+======================= ================== =============
```

Comment:
Nice, this fixes a ton of broken links you can see in the current version of the docs build: https://numpy.org/devdocs/reference/random/new-or-different.html

With this PR: https://output.circle-artifacts.com/output/job/13e9f420-b1d5-4deb-981e-d6e237ba7e91/artifacts/0/doc/build/html/reference/random/new-or-different.html